Microsoft Defender/README.md

```diff
@@ -0,0 +1,7 @@
+# Microsoft Defender
+
+## Description
+Microsoft Defender is an offer from Microsoft for individuals, business and enterprise to protect their digital assets (endpoints, accounts, emails, ...)
```

Comment:
```suggestion
Microsoft Defender for Endpoint is an enterprise endpoint security platform designed to help enterprise networks prevent, detect, investigate, and respond to advanced threats.
```

I suggest that we use the sentence from [docs.microsoft.com](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/microsoft-defender-endpoint?view=o365-worldwide)